doc/modules/groups.h

```diff
@@ -183,6 +183,16 @@
  */
 
 /**
+ * @defgroup BoundaryFluidPressures Describing fluid pressure boundary conditions 
+ * for the fluid velocity field
+ *
+ * A module for the definition of functions and classes that have to do with
+ * describing fluid pressure boundary values for the fluid velocity field.
```

Comment:
I think this is only used for the melt stuff, right? If so, maybe add a sentence to that effect.